packages/ui/src/components/pages/Request/index.tsx

```diff
@@ -15,10 +16,11 @@ import { walletInfo } from 'walletInfo';
 const Request: FC<Props> = ({ className = '' }) => {
   const [searchParams] = useSearchParams();
   const { handleWalletRequest } = useWalletState();
+  const { t } = useTranslation();
 
   if (!isChildTabOrPopup()) {
     console.error('This page should not be open directly!');
-    throw new CoongError(ErrorCode.UnknownRequestOrigin);
+    throw new CoongError(t(ErrorCode.UnknownRequestOrigin));
```

Comment:
These are raw ErrorCode, so we should not the `t` function around it. We'll do it when showing the error message to users in `InvalidRequest` view. So we should remove the `t(` here, as well as the one at line 41 and other places.